wallet/src/de/schildbach/wallet/data/BlockchainIdentityData.kt

```diff
@@ -19,24 +19,44 @@ package de.schildbach.wallet.data
 
 import androidx.room.Entity
 import androidx.room.PrimaryKey
+import org.bitcoinj.core.Coin
 import org.bitcoinj.core.Sha256Hash
-import org.bitcoinj.core.TransactionOutPoint
-import org.bitcoinj.evolution.CreditFundingTransaction
 import org.dashevo.dashpay.BlockchainIdentity
+import org.dashevo.dpp.identity.IdentityPublicKey
 
 @Entity(tableName = "blockchain_identity")
-class BlockchainIdentityData(var index: Int?,
+data class BlockchainIdentityData(var creationState: State?,
+                             var creationStateError: Boolean,
                              var username: String?,
                              var creditFundingTxId: Sha256Hash? = null,
-                             var lockedOutpoint: TransactionOutPoint? = null,
```

Comment:
Technically, we don't need to save this in the database because it is part of the credit funding transaction.

Reply:
The CreditFundingTransaction data will be most likely very useful when using this class so I think we should add it as a field that is somehow automatically loaded from the wallet when during retrieving BlockchainIdentityData from database.